web/.env-template

```diff
@@ -5,7 +5,7 @@ CYPRESS_COGNITO_USER_POOL_ID=
 CYPRESS_COGNITO_WEB_CLIENT_ID=
 API_BASE_URL=
 AUTH_DOMAIN=
-REDIRECT_URL=
+REDIRECT_URL=http://localhost:3000/loading
```

Comment:
This is the template, no need for putting values in here

Reply:
same comment for the api env template